coriolisclient/cli/migrations.py

```diff
@@ -151,12 +157,15 @@ def take_action(self, args):
         destination_environment = None
         if args.destination_environment:
             destination_environment = json.loads(args.destination_environment)
+        if args.network_map:
```

Comment:
Please declare `network_map = None` at the beginning of the method.
It's possible that someone will go like `--network-map ""` (empty string), in which case `network_map` will be undefined when used on line 168.